app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post, counter_cache: :likes_counter
+end
```

Comment:
- [ ] The Like model should have a method that updates the likes counter for a post. This should be similar to the example below:

```ruby
def update_likes_counter
  post.increment!(:likes_counter)
end
```

- **[OPTIONAL]** It is a nice idea to use one of the Rails Active Callbacks after_save to call the update_likes_counter method every time a new like has been saved to the database, then make the update_likes_counter` method private, similar to:

```ruby
  after_save :update_like_counter

  private

  def update_like_counter
    author.increment!(:likes_counter)
  end
```